zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs

```diff
@@ -777,8 +805,11 @@ mod transform_to {
     }
 }
 
-impl<F: Field> KeccakPackedConfig<F> {
-    pub(crate) fn configure(meta: &mut ConstraintSystem<F>, r: Expression<F>) -> Self {
+impl<F: Field> SubCircuitConfig<F> for KeccakCircuitConfig<F> {
+    type ConfigArgs = Expression<F>;
```

Comment:
Perhaps we can also pass in `keccak_table` as argument? Then all tables will be allocated in the top level, which might be more consistent?

And we can also remove the necessity to [allocate a dummy advice](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/925/files#diff-6a80b984a7756f4e24712f75e42b12ea3be5af4111bdbcfcc08d1b18d33d22acR57-R58) beforehand in the test of `KeccakCircuit` with challenge API.

Reply:
thanks for the suggestion!